kie-spring-boot/pom.xml

```diff
@@ -33,18 +33,21 @@
   </modules>
 
   <properties>
-    <version.spring-boot>1.5.12.RELEASE</version.spring-boot>
-    <version.cxf.jaxrs>3.1.11</version.cxf.jaxrs>
-    <version.me.snowdrop.narayana>1.0.1</version.me.snowdrop.narayana>
+    <version.org.springframework>5.1.3.RELEASE</version.org.springframework>
+    <version.org.spring-boot>2.1.1.RELEASE</version.org.spring-boot>
```

Comment:
there is 2.1.2.RELEASE which seems is in our certified version https://github.com/snowdrop/spring-boot-bom/blob/sb-2.1.x/pom.xml#L176. Was that an omission or intent to not use that version?